content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -214,6 +215,14 @@ To request a larger volume for a PVC, edit the PVC object and specify a larger
 size. This triggers expansion of the volume that backs the underlying `PersistentVolume`. A
 new `PersistentVolume` is never created to satisfy the claim. Instead, an existing volume is resized.
 
+#### CSI Volume expansion
+
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+CSI volume expansion requires enabling `ExpandCSIVolumes` feature gate and
+running external resize controller. Please see [external resize controller](https://github.com/kubernetes-csi/external-resizer/) for more details.
```

Comment:
I still don't think this makes sense for a K8s user.  This is an instruction for the CSI driver author that they need to support this sidecar in their deployment. And that's going to be dependent per driver.

Reply:
How is this a instruction for CSI driver author? This is just telling k8s admin that they need to run resizer sidecar + enable feature gate if they want CSI resizing to work. 

For exmaple - local storage documentation had detail.  Or did you mean by k8s user, not a k8s admin but someone who just uses k8s? I think that distinction is pretty fuzzy in places and I am fine with documenting an alpha feature like this. Because even a non-admin user should know why CSI resizing isn't working for their volume.

Reply:
I'm thinking in terms of a K8s end-user assuming that the driver is installed by an admin.  Even then, as an admin, they can't just blindly add the sidecar into the spec and expect resizing to work.  The driver developers need to update, test and release their driver with resizing support.

Reply:
okay sure. I have updated the docs to reflect dependency on respective CSI driver.